packages/monorepo-scripts/README.md

```diff
@@ -2,6 +2,16 @@
 
 This repository contains a few helpful scripts for working with this mono repo.
 
+#### Scripts
+
+**`yarn deps_versions`**: Since we use Lerna + yarn workspaces, shared dependencies between packages in the monorepo get hoisted to a top-level `node_modules` directory. If two packages use different versions of the same dependency however, both get installed. To avoid having many versions of a dependency installed, we try to keep dependency versions the same across packages in the monorepo. This script will list any dependencies for which we have multiple versions installed. We can then go through them and try to consolidate to a single version where possible.
+
+**`yarn find_unused_deps`**: Sometimes we accidentally leave dependencies listed in `package.json` that are no longer being used. This script finds potential dependencies that might no longer be in use. Please verify that it is no longer in use before removing, the `depcheck` package we use under-the-hood doesn't handle some TS quirks perfectly.
+
+**`yarn remove_tags`**: Our publishing script calls `lerna publish` under-the-hood. If this command fails, it might have created new versioned git tags for each package. Removing these manually is tedious, so you can also run this command instead. Before doing so, check to see if `lerna` already created the publish commit. If so, first revert that with `git reset --hard HEAD~1`, then run this command.
```

Comment:
do we need to `git push origin development -f` after doing the hard reset? I don't remember if the publish commit automatically gets pushed

Reply:
Nope, it does not push to Github **until** the end of the publish command. If it breaks before then, it only created the commit locally.